packages/demo/backend/src/services/lend.ts

```diff
@@ -60,22 +61,20 @@ export async function getMarket(
   return await verbs.lend.getMarket({ address: marketId, chainId })
 }
 
-export async function getMarketBalance(
+export async function getPosition(
   vaultAddress: Address,
   walletId: string,
   chainId: SupportedChainId,
 ): Promise<MarketBalanceResult> {
-  const verbs = getVerbs()
   const wallet = await getWallet(walletId)
 
   if (!wallet) {
     throw new Error(`Wallet not found for user ID: ${walletId}`)
   }
 
-  return verbs.lend.getMarketBalance(
-    { address: vaultAddress, chainId },
-    wallet.address,
-  )
+  return wallet.lend!.getPosition({
```

Comment:
nit: this is outside of the scope of this PR but we should create a TODO so that the `!` is not necessary and the `lend` namespace has the correct type based on the `VerbsConfig` that the `Verbs` is instantiated with